threestudio/models/guidance/zero123_unified_guidance.py

```diff
@@ -638,6 +666,7 @@ def forward(
         loss_sd = 0.5 * F.mse_loss(latents, target, reduction="sum") / batch_size
 
         guidance_out = {
+            "loss_sds": loss_sd,
```

Comment:
Magic123 uses loss_sd so it will raise an error. How about changing the config file from loss_sds to loss_sd?

Reply:
I originally keep/duplicate both the key of both loss_sd and loss_sds, To make it compatible with different models. Yes, i will consider change the cfg. Thanks